modules/transactions.js

```diff
@@ -187,6 +187,33 @@ __private.getById = function (id, cb) {
 	});
 };
 
+__private.getVotesById = function (transaction, cb) {
+	library.db.query(sql.getVotesById, {id: transaction.id}).then(function (rows) {
+		if (!rows.length) {
+			return setImmediate(cb, 'Transaction not found: ' + id);
+		}
+
+		var votes = rows[0].votes.split(',');
+		var added = [];
+		var deleted = [];
+
+		for (var i = 0; i < votes.length; i++) {
+			if (votes[i].substring(0, 1) == "+") {
```

Comment:
I getting inconsistent results, but those tests are for browsers and code is for node.js, so they are not meaningful. I would be grateful if you can you test it under node.js.